enums.h

```diff
@@ -1185,13 +1185,13 @@ enum missile_id {
     MIS_TELEPORT      = 0x1A,
     MIS_FARROW        = 0x1B,
     MIS_DOOMSERP      = 0x1C,
-    MIS_NULL_1D       = 0x1D,
+    MIS_FIREWALLA     = 0x1D,
     MIS_STONE         = 0x1E,
-    MIS_NULL_1F       = 0x1F,
+    MIS_NONE          = 0x1F, // Unused?

```

Comment:
It may be misleading that 0x1F is `_NONE`. We usually reserve this name for -1 or perhaps 0?